crates/l2/sdk/src/sdk.rs

```diff
@@ -813,10 +813,10 @@ fn bump_gas_generic_tx(tx: &mut GenericTransaction, bump_percentage: u64) {
         *max_priority_fee_per_gas = (*max_priority_fee_per_gas * (100 + bump_percentage)) / 100;
     }
     if let Some(max_fee_per_blob_gas) = &mut tx.max_fee_per_blob_gas {
-        let factor = 1 + (bump_percentage / 100) * 10;
+        let factor = 100 + bump_percentage;
         *max_fee_per_blob_gas = max_fee_per_blob_gas
             .saturating_mul(U256::from(factor))
-            .div(10);
+            .div(100);
```

Comment:
To keep the same semantics without the integer truncation issue what you actually need is:
```suggestion
if let Some(max_fee_per_blob_gas) = &mut tx.max_fee_per_blob_gas {
        let factor = 100 + bump_percentage * 10;
*max_fee_per_blob_gas = max_fee_per_blob_gas
.saturating_mul(U256::from(factor))
            .div(1000);
```
I don't know why the magic multiply and divide by 10 was there, but if there was a reason you'll want to keep it.